public/backend/js/backend.js

```diff
@@ -1,14 +1,14 @@
 $('div.alert').delay(time).slideUp();
-
-$(document).ready(function () {
-    $('#myTable').DataTable();
+$(document).ready(function(){
+    var frmId = $('table').attr('id');
+    $('#' + frmId).DataTable();
```

Comment:
[rank C] Do you understand what I mean? I mean we need to declare id table element can readable and easy to understand. For example: 
- Your table is listing businesses, so we can set id = "list_business_table" or something like that. (instead set it as "myTable" because it's meaningless)